### PR TITLE
{math}[GCCcore/13.2.0] GDB v13.2, MPC v1.3.1, ISL v0.26, makeinfo v7.1

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen2

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/f2ba1d4b181a024411ab6bbc03f95ba7 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bwd-rockylinux-92 - Linux Rocky Linux 9.2 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 535.104.12, Python 3.9.16
See https://gist.github.com/SebastianAchilles/cab177ca23da2ead2285c6dd2f552557 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 6 out of 6 (4 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/9216a1a45b486e3f460e7630a3f3cfc3 for a full test report.

File: easybuild/easyconfigs/g/GDB/GDB-13.2-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+configopts = '--with-system-zlib --with-system-readline --with-expat=$EBROOTEXPAT '
+configopts += '--with-python=$EBROOTPYTHON/bin/python --with-isl=$EBROOTISL --with-mpc=$EBROOTMPC '
+configopts += '--enable-tui --enable-plugins --disable-install-libbfd '
+
+sanity_check_paths = {
+    'files': ['bin/gdb', 'bin/gdbserver'],
+    'dirs': [],
+}
+
```
Comment (verdurin) [inline easybuild/easyconfigs/g/GDB/GDB-13.2-GCCcore-13.2.0.eb:43]: @Micket would make sense to re-instate the `sanity_check_commands` from previous versions?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/GDB/GDB-13.2-GCCcore-13.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/g/GDB/GDB-13.2-GCCcore-13.2.0.eb:43]: @verdurin Only one `GDB` easyconfig has `sanity_check_commands` currently (`GDB-13.2-GCCcore-12.2.0.eb`), so we can/should fix this consistently across the most recent GDB easyconfigs imho...

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @Micket!

File: easybuild/easyconfigs/g/GDB/GDB-13.2-GCCcore-13.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/g/GDB/GDB-13.2-GCCcore-13.2.0.eb:43]: @verdurin see #19102

**Pull request merged**